### PR TITLE
feat(resolver): adds component resolver for VueUse components

### PR DESCRIPTION
**Pull request by wheatjs**
Added a resolver for vueuse components.

### CODE REVIEW

File: src/resolvers/vueuse.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { ComponentResolver } from '../types'
+
+/**
+ * Resolve for VueUse Components
+ * 
+ * @link https://github.com/vueuse/vueuse
+ */
+export const VueUseComponentsResolver = (): ComponentResolver => (name: string) => {
+  if (name.toLowerCase().startsWith('use') || name.toLowerCase().startsWith('on'))
```
Comment (antfu) [inline src/resolvers/vueuse.ts:9]:
```suggestion
  if (/^(?:Use|On)[A-Z]/.test(name))
```

Tho I still thinking maybe we should read the `indexes.json` to have more accurate match.

Comment (wheatjs) [reply src/resolvers/vueuse.ts:9]: Oh yeah that would be a much better way I think. Any preferred way we should go about reading `indexes.json`?

Comment (antfu) [reply src/resolvers/vueuse.ts:9]: Maybe we should ship it along with the package

Comment (antfu) [reply src/resolvers/vueuse.ts:9]: This should do in 5.0.0-beta.3:

```js
require('@vueuse/core/indexes.json')
```

**Pull request merged**